NEWS

```diff
@@ -11,9 +11,93 @@
 ### Features:
 ### Bugfixes:
 
-## 1.17.0-rc1 (May 16, 2024)
-TBD
-
+## 1.17.0-rc2 (May 16, 2024)
+### Features:
+#### UCP
+* Improved performance calculation for rendezvous protocol
+* Enabled short protocol for non-host memory types on empty messages
+* Improved performance estimation for empty messages by removing non-relevant overheads
+* Added RMA_ZCOPY_MAX_SEG_SIZE configuration parameter to allow modifying segment size for RMA-ZCOPY protocols
+* Added support for separate intra/inter-node rendezvous thresholds
+* Added support for minimal fragment size in rendezvous protocol
+* Added support for resetting request during send operation
+* Added UCX_PROTO_OVERHEAD configuration variable to allow setting protocol overheads
+* Improved performance for combined Active Message/RMA scenarios by separating them to different lanes
+* Added support for device staging buffers in pipeline protocols
+* Enabled ODP on Grace platforms by default 
+#### RDMA CORE (IB, ROCE, etc.)
+* Introduced the UCX_REVERSE_SL environment variable to configure reverse SL for DC transport. By default, it uses UCX_IB_SL.
```

Comment:
What is reverse SL ?

Reply:
@shamisp 
That's a HW feature, documented in other places. Are you sure more details are needed here, in addition to what is already mentioned?

Reply:
We need to use here a description that external users can understand. I don't believe anybody will know what is UCX_REVERSE_SL

Reply:
How about:
"Introduced the UCX_REVERSE_SL environment variable to configure priority level for DC transport inbound traffic."

Reply:
I like @yosefe  proposal

Reply:
Actually, current version is fine as well. I like that it calls out default behaviour.